test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -6,575 +6,596 @@
 using System.IO;
 using System.Linq;
 using Microsoft.AspNet.Testing.xunit;
-using Microsoft.Dnx.CommonTestUtils;
+using Microsoft.Dnx.Testing;
+using Newtonsoft.Json.Linq;
 using NuGet;
 using Xunit;
 
 namespace Microsoft.Dnx.Tooling
 {
-    public class DnuPackTests
+    public class DnuPackTests : DnxSdkFunctionalTestBase
```

Comment:
I'm on board, but `misc` is a terrible name for a folder that contains projects required by tests...